src/components/side-by-side-rendered-diff.jsx

```diff
@@ -29,6 +31,21 @@ export default class SideBySideRenderedDiff extends React.Component {
       transformInsertions = showAdditions;
     }
 
+    if (this.props.fromUuid === this.props.toUuid) {
```

Comment:
I kinda wonder if this part would be better raised up into `DiffView` (so the case where this view gets rendered from the Switch of Doom™ to return an `InlineRenderedDiff` instead of a `SideBySideRenderedDiff`. Or something else entirely.

*Generally,* it’s a good practice to use a different type of behavior/component/view for a given circumstance than to add conditions to an existing one. It tends to help keep things from getting overcomplicated or hard to tease apart when you need to refactor.

Reply:
Thanks, that makes sense!

I wanted to use `SandboxedHtml` instead of `InlineRenderedDiff` because this was used when both versions are the same and `InlineRenderedDiff` would show the the `No changes` message (excepting this [issue](https://github.com/edgi-govdata-archiving/web-monitoring-processing/issues/148). But I see how I can just pull that out into `DiffView` now.

This does make me wonder what analysts expect in this case. I'll pose a question on Slack so analysts start thinking about it. Basically, 'Is being able to choose the same version something they want/expect to do? What feedback do they expect from that choice?'

I can remove this line of code until we have a better understanding.